scripts/dev/src2asciidoc.py

```diff
@@ -498,11 +499,12 @@ def _format_block(filename, what, data):
 
 def regenerate_manpage(filename):
     """Update manpage OPTIONS using an argparse parser."""
-    # pylint: disable=protected-access
     parser = qutebrowser.get_argparser()
     groups = []
     # positionals, optionals and user-defined groups
+    # pylint: disable=protected-access
     for group in parser._action_groups:
+    # pylint: enable=protected-access
```

Comment:
Sorry, my bad - it's actually needed for the `for action in group._group_actions:` below too. So I think it's fine to just put it after the whole `for` block. I'll fix this up while merging though!